packages/protocol/script/DeployOnL1.s.sol

```diff
@@ -245,6 +245,20 @@ contract DeployOnL1 is DeployCapability {
         copyRegister(rollupAddressManager, _sharedAddressManager, "signal_service");
         copyRegister(rollupAddressManager, _sharedAddressManager, "bridge");
 
+        deployProxy({
+            name: "mainnet_taiko",
```

Comment:
Do we still need the TaikoL1 deployment below?

Reply:
Maybe not, but TaikoL1 is "cleaner" (without the address caching), so I keep using it in case the contact is actually called. MainnetTaikoL1 cannot be invoked as the cached addresses don't have code deployed.